packages/synapse-react-client/src/components/GoalsV2/GoalsV2.tsx

```diff
@@ -119,18 +121,18 @@ export const GoalsV2: React.FC<GoalsV2Props> = (props: GoalsV2Props) => {
         })}
       />
       {goalError && <ErrorBanner error={goalError} />}
-      <div className={`Goals${showDesktop ? '__Desktop' : ''}`}>
-        {goalsDataArray.map((row, index) => {
-          return showDesktop ? (
-            <div>
-              <GoalsV2Desktop key={index} {...row} />
-            </div>
-          ) : (
-            <Box sx={{ display: 'grid' }}>
-              <GoalsV2Mobile key={index} {...row} />
-            </Box>
-          )
-        })}
+      <div className={`Goals`}>
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexDirection: showDesktop ? 'row' : 'column',
+          }}
+        >
```

Comment:
Updated so classnames match Goals v1 changes. `flex` works fine for both desktop and mobile here, so just use that.